explorer/client/src/components/pagination/Pagination.tsx

```diff
@@ -1,178 +1,254 @@
 // Copyright (c) 2022, Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import cl from 'classnames';
 import { memo, useState, useCallback, useEffect } from 'react';
 
-import { ReactComponent as ContentForwardArrowDark } from '../../assets/SVGIcons/forward-arrow-dark.svg';
+import TabFooter from '../../components/tabs/TabFooter';
```

Comment:
Why add the tab footer to the pagination module

Reply:
This fixed an issue with alligning the text across the footer.

Reply:
They are separate items, there are multiple areas where the tap footer is not path of the pagination

Reply:
<img width="742" alt="Screen Shot 2022-07-20 at 3 14 53 PM" src="https://user-images.githubusercontent.com/8676844/180063772-2b67d4a5-02b5-47b2-b6e3-ac7fd728fe21.png">
<img width="664" alt="Screen Shot 2022-07-20 at 3 15 07 PM" src="https://user-images.githubusercontent.com/8676844/180063806-8954ee78-8e7f-49b7-87f0-ccb8a6795389.png">

Reply:
I've removed `TabFooter` from Pagination.